.drone.yml

```diff
@@ -50,7 +50,7 @@ depends_on:
   - test
 
 trigger:
-  branch: main
+  branch: add-pkce-auth
```

Comment:
These drone file changes need to be cleaned up. If we want to add a publish stage for pushes that seems better than editing the drone file to have feature branch builds.